### PR TITLE
Revert unrelated change from #24878

### LINKED ISSUE #26295: GAME: Error: /obj/item/weapon/gun/ballistic/minigun([0x202c7cf]) took longer then 1 second to delete (took 809 seconds to delete)
I have found this in the log files

https://tgstation13.org/parsed-logs/basil/logs/2017/04-April/17-Monday.txt (control+f for `08:21:55`)

@tgstation/commit-access 

There are a few more. https://tgstation13.org/parsed-logs/deletelagfind.txt

This happened after a bagil "crash" only one that nobody was around to fix, so it resolved itself manually

### PR DESCRIPTION
**Pull request by MrsTonedOne**
fixes #26295 (I think, more confirmation required, but its 3am)

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/laser_gatling.dm [hunk 1]
```diff
@@ -110,8 +110,7 @@
 	casing_ejector = 0
 	var/obj/item/weapon/minigunpack/ammo_pack
 
-/obj/item/weapon/gun/ballistic/minigun/Initialize(mapload)
-	..()
+/obj/item/weapon/gun/ballistic/minigun/New()
```
Comment (coiax) [inline code/modules/projectiles/guns/ballistic/laser_gatling.dm]: not calling ..() in New?

Comment (MrStonedOne) [inline code/modules/projectiles/guns/ballistic/laser_gatling.dm]: Look below mang

**Comment by coiax** [comment]
It might qdel in New, and that would make Cyberboss sad. Although I'm still unclear why qdelling in New is bad.

**Comment by MrStonedOne** [comment]
its better then it taking 800 seconds to delete.

**Pull request merged**